source/_integrations/renson.markdown

```diff
@@ -0,0 +1,19 @@
+---
+title: Renson
+description: Instructions on how to integrate Renson Endura Delta sensors into Home Assistant.
+ha_category:
+  - Sensor
+  - Binary Sensor
+ha_release: 2021.10
+ha_iot_class: Local Poll
+ha_config_flow: true
+ha_codeowners:
+  - '@jimmyd-be'
+ha_domain: integration
+ha_platforms:
+  - sensor
+---
+
+The Reason integration pulls in data from the Renson Endura delta application for Android en iOS.
```

Comment:
This sounds like the integration interfaces with the app on the phone, but that's not what it does, right?

Reply:
Changed the text a bit.